00-Raster-Structure.Rmd

```diff
@@ -373,7 +387,7 @@ plotRGB(RGB_stack, r = 1, g = 2, b = 3,
 
 ```
 
-Below - the black edges have been assigned NoDataValues (`NA`). R doesn't render 
+Below - the black edges have been assigned NoDataValues (`NA`). R doesn't render
```

Comment:
We should be consistent. I forget if the tif tag is `NoData` or `NoDataValue` but whatever it is, let's use one and then always render it w/ inline code it given it's a tag.